actions/compatibility/entrypoint/main.go

```diff
@@ -0,0 +1,296 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+
+	msemver "github.com/Masterminds/semver"
+	"github.com/blang/semver"
+	"github.com/cloudfoundry/buildpacks-ci/tasks/cnb/helpers"
```

Comment:
I am going to knitpick at this a little as well sorry. I would like to be able to remove this dependency if at all possible. Is there anything that we could do to make accessing the `RuntimeToSDK` table easier using `cargo` libraries that would allow you to switch away from this helper?

Reply:
this is a good idea. I will work some stuff around

Reply:
done in https://github.com/paketo-buildpacks/github-config/pull/351/commits/24f707e77d164975cf56366b116eae018be1ba8b